test/gtest/uct/test_md.cc

```diff
@@ -817,14 +817,14 @@ UCS_TEST_SKIP_COND_P(test_md, dereg_bad_arg,
 UCS_TEST_SKIP_COND_P(test_md, exported_mkey,
                      !check_caps(UCT_MD_FLAG_EXPORTED_MKEY))
 {
-    static const size_t size       = 1 * UCS_MBYTE;
-    static const unsigned md_flags = UCT_MD_MEM_ACCESS_REMOTE_PUT |
-                                     UCT_MD_MEM_ACCESS_REMOTE_GET;
+    size_t size = ucs::rand() % UCS_MBYTE;
     uct_mem_h export_memh;
+    void *address;
     ucs_status_t status;
 
-    std::vector<uint8_t> mem_buffer(size);
-    status = reg_mem(md_flags, mem_buffer.data(), size, &export_memh);
+    status = ucs_mmap_alloc(&size, &address, 0, "test_md_exp_mkey");
```

Comment:
print the size with UCS_TEST_MESSAGE so if it fails we could repro

Reply:
ok.
actually I did it initially, but it does not help) It fails sporadically with the same message size